darwin/dataset/local_dataset.py

```diff
@@ -501,7 +501,11 @@ def get_annotation_filepaths(
     """
 
     if partition is None:
-        return (str(e) for e in sorted(annotations_dir.glob("**/*.json")))
+        return (
+            str(e)
+            for e in sorted(annotations_dir.glob("**/*.json"))
+            if "/.v7/" not in str(e)
```

Comment:
```
            if "/.v7/" not in str(e)
```

Do we need this part here? 🤔 as `glob("**/*.json")` is anyways fetching only the json files?

Reply:
I think we do sadly, and I can reproduce this by instantiating a `LocalDataset` - The `**` in `glob()` is recursively checking the annotations directory. Since it now will contain `.v7/metadata.json`, it picks this file up unless we tell it to ignore filepaths containing `/.v7/`

The code I used to reproduce this is below - Before running it, a dataset release has to be pulled using 0.8.59 so that the metadata file is present in the annotations directory
```
from pathlib import Path

from darwin.dataset import LocalDataset

my_dataset = LocalDataset(
    dataset_path=Path("/path/to/dataset"),
    annotation_type="bounding_box",
)

```